### PR TITLE
packaging

### PR DESCRIPTION
**Pull request by paulrouget**
This is the first step for packaging browser.html.

Running `npm run package` will creates a `dist` directory with the necessary files and concatenated scripts. Files are not minified yet (can't simply minify ES6 scripts).

For this script to work, this change is required for now:
```diff
diff --git a/src/service/history.js b/src/service/history.js
index 5cf07da..30e1dcf 100644
--- a/src/service/history.js
+++ b/src/service/history.js
@@ -34,7 +34,8 @@ define((require, exports, module) => {
   exports.PageQuery = PageQuery;

   const service = address => {
-    const worker = require('common/worker!service/history-worker');
+    return () => {};
+    // const worker = require('common/worker!service/history-worker');

     worker.onmessage = ({data: {type, action}}) => {
       if (type === 'PageResult') {
```

### CODE REVIEW

**Comment by paulrouget** [comment]
Follow up: #522

**Comment by gordonbrander** [comment]
@Gozala @paulrouget should we wait until this history-worker thing is fixed?

File: build.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+({
+  baseUrl: 'node_modules/',
+  name: '../main',
+  optimize: 'none', // uglify doesn't work with ES6
+  paths: {
+    browser: '../src/browser',
+    browser2: '../src/browser2',
```
Comment (Gozala) [inline build.js]: What is `browser2` ?

**Comment by paulrouget** [comment]
Happening in #547.

**Pull request closed without merging**